### ISSUE TITLE
podman network create: add option to set custom network driver settings, eg like MTU

### ISSUE THREAD

**Issue by pschiffe** [containers/podman#8454]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**

Docker command `docker network create` has the `-o, --opt map              Set driver specific options (default map[])` option.

With podman, one have to modify the config file in `/etc/cni/` once the network is created to change its settings, which is not very automation friendly.

Related ansible collection RFE: https://github.com/containers/ansible-podman-collections/issues/147

**Comment by rhatdan** [comment]
@pschiffe interested in opening a PR to fix this?

**Comment by afbjorklund** [comment]
This would be useful for minikube as well, I can probably do a PR for it

**Comment by rhatdan** [comment]
You got it.

**Comment by afbjorklund** [comment]
Most likely it will be a simple option to "network create", rather than -o com.docker.network.driver.mtu=1500

**Comment by afbjorklund** [comment]
Seems to be working:

```
$ sudo ./bin/podman network create podman1
/etc/cni/net.d/podman1.conflist
$sudo ./bin/podman network create podman2 --mtu 5000
/etc/cni/net.d/podman1.conflist
```

27: cni-podman1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu **1500** qdisc noqueue state UP group default qlen 1000

29: cni-podman2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu **5000** qdisc noqueue state UP group default qlen 1000

**Comment by pschiffe** [comment]
Looking great and good enough for me, thank you!

**Comment by afbjorklund** [comment]
Syntax now changed to:

`podman network create podman2 --opt mtu=5000`

**Comment by pschiffe** [comment]
Resolved via #8457
